common/spec/spec_helper.rb

```diff
@@ -59,6 +60,14 @@
     config.debug_logger = File.open(ENV["DEPENDABOT_TEST_DEBUG_LOGGER"], "w")
   end
 
+  # Prevent auth headers and username:password params being written to VCR cassets
+  config.before_record do |interaction|
+    interaction.response.headers.transform_keys!(&:downcase).delete("set-cookie")
+    interaction.request.headers.transform_keys!(&:downcase).delete("authorization")
+    uri = URI.parse(interaction.request.uri)
+    interaction.request.uri.sub!(%r{:\/\/.*#{Regexp.escape(uri.host)}}, "://#{uri.host}")
+  end
```

Comment:
Snuck this in part of this PR as a separate commit as the VCR cassette included an authorization header with a PAT.

Reply:
Nice one, did you check to make sure there are no other PATs in the VCR records?

Do we _need_ these requests to be authenticated? I think we should still keep this in, but we can maybe also record them without being auth'ed?